client/modules/core/startup.js

```diff
@@ -2,7 +2,12 @@ import store from "amplify-store";
 import { Meteor } from "meteor/meteor";
 import { Tracker } from "meteor/tracker";
 import { Accounts } from "meteor/accounts-base";
+
 import Reaction from "./main";
+import Logger from "/client/modules/logger";
```

Comment:
I think we want to import `Reaction` and `Logger` from `.client/api` like we do elsewhere for consistency.